inference-engine/tests/functional/plugin/cpu/single_layer_tests/reduce_ops.cpp

```diff
@@ -155,186 +240,424 @@ const std::vector<ngraph::helpers::ReductionType> reductionLogicalTypes = {
         ngraph::helpers::ReductionType::LogicalAnd
 };
 
-const std::vector<std::vector<size_t>> inputShapes = {
-        std::vector<size_t>{10, 5, 15, 12},
-        std::vector<size_t>{3, 5, 7, 9},
+std::vector<std::vector<ov::test::InputShape>> inputShapes = {
+    {{{}, {{2, 19, 2, 9}}}},
+    {{{-1, {2, -1}, -1, -1}, {{2, 9, 2, 9}, {2, 19, 2, 9}, {2, 9, 2, 15}}}},
+    {{{{1, 5}, {2, 5}, {1, 5}, {1, 20}}, {{3, 2, 3, 2}, {3, 3, 3, 2}, {3, 2, 3, 3}}}}
+};
+
+std::vector<std::vector<ov::test::InputShape>> inputShapes_5D = {
+    {{{}, {{2, 19, 2, 2, 9}}}},
+    {{{-1, {2, -1}, -1, -1, -1}, {{2, 9, 2, 2, 9}, {3, 9, 2, 2, 9}, {2, 9, 2, 5, 9}}}},
+    {{{{1, 5}, {2, 5}, {1, 5}, {1, 5}, {1, 5}}, {{2, 3, 2, 3, 2}, {3, 3, 2, 3, 2}, {2, 3, 2, 4, 2}}}}
+};
+
+std::vector<std::vector<ov::test::InputShape>> inputShapes_6D = {
+    {{{}, {{2, 3, 2, 2, 2, 9}}}},
+    {{{-1, {2, -1}, -1, -1, -1, -1}, {{2, 3, 2, 2, 2, 9}, {2, 3, 3, 2, 2, 9}, {2, 3, 2, 2, 2, 11}}}},
+    {{{{1, 5}, {2, 5}, {1, 5}, {1, 5}, {1, 5}, {1, 5}}, {{2, 3, 2, 2, 3, 2}, {2, 3, 3, 2, 3, 2}, {2, 3, 2, 2, 3, 3}}}}
 };
 
 std::vector<CPUSpecificParams> cpuParams_4D = {
         CPUSpecificParams({nChw16c}, {nChw16c}, {}, {}),
-        CPUSpecificParams({nchw}, {nchw}, {}, {})
+        CPUSpecificParams({nchw}, {nchw}, {}, {}),
```

Comment:
Could you please elaborate why `nChw8c` layout is not tested?

Reply:
The function filterCPUSpecificParams assures nChw8c to be included. Thanks Maksim for the comment.